group_vars/all.yml

```diff
@@ -70,7 +70,7 @@ oracle_rel: "{{ lookup('env','ORA_RELEASE') | default('latest',true) }}"
 oracle_edition: "{{ lookup('env','ORA_EDITION') | default('EE',true) }}"
 cluster_type: "{{ lookup('env','CLUSTER_TYPE') | default('NONE',true) }}"
 
-container_db: "{{ lookup('env','ORA_DB_CONTAINER') | lower | default('false',true) }}"
+container_db: "{{ lookup('env','ORA_DB_CONTAINER') | lower | default('false',true) | bool }}"
```

Comment:
This change appears to negate the default:  `'false' | bool` in jinja evaluates to true.  

Consider the logic in https://github.com/google/oracle-toolkit/pull/326/commits/2b146c9788d694ab8e6fbb050efbefe0a1a3cb25#diff-7be6987d456ad921f461608b488a7593f037ba8fc6e43ed862c903e91109c00aL74 instead

```jinja
container_db: "{{ lookup('env','ORA_DB_CONTAINER') | default(false,true) | bool }}"
```